wdl-format/tests/format/funky_command_leading_whitespace/source.formatted.wdl

```diff
@@ -0,0 +1,28 @@
+## This is a test of leading whitespace stripping and normalization in command blocks.
+
+version 1.2
+
+task no_whitespace {
+    command <<<
+        echo "hello"
+    >>>
+}
+
+task short_leading_whitespace {
+    command <<<
+        echo "hello"
+    >>>
+}
+
+task long_leading_whitespace {
+    command <<<
+        echo "hello"
+    >>>
+}
+
+task mixed_leading_whitespace {
+    command <<<
+        echo "hello"
+              echo "world"
```

Comment:
This makes sense based on what we've discussed previously, but I don't particularly like the result.

Reply:
How would you ideally format this? I think the only way to improve it is to actually parse the Bash to determine that extra whitespace is meaningless. And I've gone on record as saying parsing Bash is a barrel of worms we shouldn't open.